builder/FindPackages.cmake

```diff
@@ -488,6 +493,7 @@ if( Linux )
   # optional:
   pkg_check_modules( PKG_X11       x11 )
   pkg_check_modules( PKG_LIBVA_X11 libva-x11>=0.33 )
+  pkg_check_modules( PKG_MFX       mfx>=1.28 )
```

Comment:
better to use "libmfx" instead of "mfx" since the later on is a symlink, i.e. legacy back compatibility...

Reply:
ok

Reply:
so, you will fix that?